src/vm/fieldmarshaler.h

```diff
@@ -463,6 +478,7 @@ class FieldMarshaler
     RelativeFixupPointer<PTR_FieldDesc> m_pFD;      // FieldDesc
     UINT32           m_dwExternalOffset;    // offset of field in the fixed portion
     NStructFieldType m_nft;
+    NativeFieldCategory m_nfc;
```

Comment:
Ah ha! This is the reason for the increase in `MAXFIELDMARSHALERSIZE`. Please try and use `static_assert()` to validate this.

Reply:
Oh. Yay! All good then.